steps/email.ts

```diff
@@ -0,0 +1,19 @@
+import { isAfter, parseISO, subMinutes } from 'date-fns'
+import { NotifyClient } from 'notifications-node-client'
+import { expect } from '@playwright/test'
+
+export const verifyEmailSent = async (subject: string, emailAddress: string) => {
+  const client = new NotifyClient(process.env.NOTIFY_API_KEY)
+
+  const response = await client.getNotifications('email')
+  const { notifications } = response.data
+
+  const matchedNotification = notifications.find(
+    notification =>
+      isAfter(parseISO(notification.sent_at), subMinutes(Date.now(), 1)) &&
```

Comment:
Would it be worth abstracting this into a separate method for readability? Is it essentially `notificationWasSentInTheLastMinute`?